model/SearchEngine/Driver/Elasticsearch/ElasticSearchConfig.php

```diff
@@ -47,48 +55,69 @@ public function __construct(ServiceOptionsInterface $serviceOptions)
     public function getHosts(): array
     {
         $hosts = [];
-        foreach ($this->serviceOptions->get(self::class, self::OPTION_HOSTS) as $host) {
-            if (is_array($host) && isset($host['host'])) {
-                $hosts[] = ($host['scheme'] ?? 'http') . '://' . $host['host'] . ':' . ($host['port'] ?? 9200);
-            } else {
-                $hosts[] = $host;
+        if ($this->serviceOptions->get(self::class, self::OPTION_HOSTS)) {
```

Comment:
I would recommend to do a small extension of the unit test, just setting the env vars like: 

$_ENV['MY_ENV_VAR'] = 'xxx'

Just to make sure nobody will remove it. I think it is really small test method